legislative/index.json

```diff
@@ -147,332 +147,332 @@
     ]
   },
   {
-    "comment": "Legislative Assembly of Amazonas",
-    "house_item_id": "Q6518202",
-    "position_item_id": "Q53864935",
+    "comment": "Municipal Chamber of Belém",
+    "house_item_id": "Q53934974",
```

Comment:
Just another reordering:

```bash
$ cat legislative/index.json | jq '[ .[].house_item_id ] | sort' > a.json
$ git show master:legislative/index.json | jq '[ .[].house_item_id ] | sort' > b.json
$ diff a.json b.json
$ cat legislative/index.json | jq '[ .[].position_item_id ] | sort' > a.json
$ git show master:legislative/index.json | jq '[ .[].position_item_id ] | sort' > b.json
$ diff a.json b.json
$ cat legislative/index.json | jq '[ .[].terms[].term_item_id ] | sort' > a.json
$ git show master:legislative/index.json | jq '[ .[].terms[].term_item_id ] | sort' > b.json
$ diff a.json b.json
34d33
<   "Q53934889",
40a40
>   "Q53934997",
```

The term ID change is the result of a term item merge: https://www.wikidata.org/w/index.php?title=Q53934997&redirect=no